libs/wingsdk/test/target-sim/queue.test.ts

```diff
@@ -2,7 +2,7 @@ import * as cloud from "../../src/cloud";
 import { SimApp, Testing } from "../../src/testing";
 import { listMessages } from "./util";
 
-jest.setTimeout(10_000); // 5 seconds
```

Comment:
How about we change the default test timeout for all SDK tests? (requires changing projen file) We could probably just add `--testTimeout=300000` to the command

Reply:
https://github.com/winglang/wing/issues/1120